webapp/src/main/resources/properties/en.properties

```diff
@@ -370,6 +370,18 @@ drops.controlbar.button.tooltip.import=Import Request
 # Tooltip on cancel drop button
 drops.controlbar.button.tooltip.cancel=Cancel Request
 
+# Title of the confirmation dialog box when a user attempts to cancel a review or translation request
+drops.confirm.cancel.model.title=Really Cancel?
```

Comment:
Can we replace "Really Cancel?" with "Confirm cancel"? (to be consistent with the Confirm delete dialog in workbench).
<img width="1462" alt="confirmdelete" src="https://user-images.githubusercontent.com/17402500/42599313-b02baa10-8513-11e8-8820-44a224cb5fe3.png">